### PR TITLE
[2.3] Slugify deprecation

### PR DESCRIPTION
**Pull request by Koc**
See https://github.com/sonata-project/SonataCoreBundle/pull/214#issuecomment-160592081

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
I don't all understand. Why are you deprecated the `NativeSuglify` to allow 2.0 version of `cocur/slugify`?

**Comment by OskarStark** [comment]
Because we use the own implementation to stay BC. But this should neremoved
in the Future



Am Montag, 30. November 2015 schrieb Sullivan SENECHAL :

**Comment by Koc** [comment]
@Soullivaneuh I'm have project where want to use `cocur/slugify: ~2.0`. But I cannt't do this because SonataCoreBundle blocks version. The first commit allows this version.

As you can see from d53899c24be7341b9e0e1b4dfcffa71c238609e0 commit - this class was deprecated when it was added in the bundle ;). So I just added deprecation notice.

At the same time, there is no any usage of this class or twig filter nowhere in the SonataAdminBundle or SonataDoctrineOrmAdminBundle, so it can be safety removed in the future. I propose merge this PR into master and create `2.3` branch of the bundle based on master after merge, and them start working on `3.0` in master.

It absolutley safe. People who using deprecated classes or services would stay on 2.3 version.

**Comment by Soullivaneuh** [comment]
Ok seems logic. But are you sure this class is deprecated since `2.3`? Ping @rande.

**Comment by Koc** [comment]
Don't know. Does it matter?

**Comment by Soullivaneuh** [comment]
For consistency, yes! ;-)

**Comment by OskarStark** [comment]
an UGRADE NOTICE is needed

**Comment by Koc** [comment]
PR is ready. Travis pass tests. Please review and merge. I've updated the docs, but if I do it wrong - please fix it by yourself.

//cc @OskarStark @rande @Soullivaneuh

**Comment by OskarStark** [comment]
for me this is RTM :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ A [BC BREAK] means the update will break the project for many reasons :
 * new dependencies
 * class refactoring
 
+### 2015-12-12
+
+* The services `sonata.core.slugify.cocur` and `sonata.core.slugify.native` are deprecated.
+* The Twig filter `sonata_slugify` are deprecated. Install `cocur/slugify` and enable `CocurSlugifyBundle` https://github.com/cocur/slugify#symfony2 for using `slugify` filter.
```
Comment (OskarStark) [inline CHANGELOG.md]: are -> is ?

Comment (Koc) [inline CHANGELOG.md]: @OskarStark fixed

**Comment by OskarStark** [comment]
The travis error is not related to this PR

@rande please merge this :+1:

**Comment by Koc** [comment]
Any reason not to merge this?

**Comment by Koc** [comment]
@rande @OskarStark @Soullivaneuh please merge. I cann't use slugify 2.0 without this PR

**Comment by OskarStark** [comment]
Sorry @Koc 

Thank you :+1:

**Pull request merged**